cognitedata/reveal
# feat: add information and capability to update assetRef field for annotations in point clouds in Reveal examples

#### Type of change
![Feat](https://img.shields.io/badge/Type-Feat-green)  


#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-664

## Description :pencil:
Added simple UI for point cloud annotations for debugging purposes. It includes basic information about the annotation and also allows to change `assetRef` field.

<img width="1810" alt="Screenshot 2023-01-05 at 10 39 51" src="https://user-images.githubusercontent.com/45129444/210751509-dfeb83ee-7fc8-4e1b-a173-25712ceabe9b.png">




## How has this been tested? :mag:
Tested on point cloud model with annotations. 



## Test instructions :information_source:
Open any point cloud model that have annotations, click on the annotation and open "Point cloud objects" -> "Selected annotation" menu section. You will see all the information, you can try updating assetRef field to some value and then clicking again at the same annotation to see whether the value was updated properly.



## Checklist :ballot_box_with_check:
